sansa-ml-spark/src/main/scala/net/sansa_stack/ml/spark/clustering/algorithms/DBSCAN.scala

```diff
@@ -68,7 +68,7 @@ class DBSCAN() extends Serializable {
         pointRDD.analyze()
 
         // Perform Spatial Partitioning with QuadTree
-        pointRDD.spatialPartitioning(GridType.QUADTREE, 16)
+        pointRDD.spatialPartitioning(GridType.QUADTREE, 2)
 
         // val boundaryEnvelopes = pointRDD.getPartitioner.getGrids
```

Comment:
In case these commented out lines are not needed anymore, consider removing them!